dts/bindings/display/nrf-led-matrix.yaml

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) 2021, Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+description: Generic LED matrix driven by nRF SoC GPIOs
+
+compatible: "nrf-led-matrix"
+
+include: display-controller.yaml
+
+properties:
+    row-gpios:
+      type: phandle-array
+      required: true
+      description: |
+        Array of GPIOs to be used as rows of the matrix.
+
+    col-gpios:
+      type: phandle-array
+      required: true
+      description: |
+        Array of GPIOs to be used as columns of the matrix.
+
+    pixel-mapping:
```

Comment:
This property could be optional on "regular" LED matrices (like the one in V2). I guess this exists due to the 3x9 matrix used as 5x5 in V1.

Reply:
Yes, for "regular" matrices, I could generate such array in the driver. But isn't it good to have the mapping specified explicitly in DTS so that everything regarding the hardware is clear?

Reply:
fair enough